src/SubNav.jsx

```diff
@@ -0,0 +1,29 @@
+import React, {useState} from 'react'
+import {UnderlineNav} from '@primer/react'
+
+export function SubNav() {
+  return (
+    <UnderlineNav
+      aria-label="Page navigation"
+      sx={{position: 'sticky', top: '66px', background: '#ffffff', borderBottom: 'none'}}
+    >
+      <UnderlineNav.Link href="#usage" selected={location.hash === '#usage'}>
```

Comment:
Oh sorry I couldn't help you with this selected state! I think I was so focused on "why this component doesn't have onSelect prop" and completely missed the fact that we can always manually do it 🙈  I am glad it is all sorted out though 😊 And this looks amazing 🔥

Reply:
@broccolinisoup no worries at all! If you want to use this branch to test the newer UnderlineNav when its shipped definitely feel free! 🚀

Reply:
@langermank Thank you! I will 🤩